pkg/virtctl/console/console.go

```diff
@@ -74,6 +74,11 @@ func (c *Console) Run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	value := os.Getenv("KUBECTL_PLUGINS_CURRENT_NAMESPACE")
```

Comment:
You could use LookupEnv as in https://github.com/kubevirt/kubevirt/pull/1361/files#diff-0c40f15794fc965becd5de8b54cd4da9R88